src/screens/NewsScreen/index.js

```diff
@@ -7,7 +7,7 @@ import {
   StatusBar,
   StyleSheet
 } from 'react-native';
-import {SafeAreaProvider} from 'react-native-safe-area-context';
+import {SafeAreaProvider, SafeAreaView} from 'react-native-safe-area-context';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include `SafeAreaView` from 'react-native-safe-area-context'. This is a good change as it provides a way to render content within the safe area boundaries of a device. It's important to ensure that all parts of the application respect the safe areas to prevent UI elements from being clipped or hidden by rounded corners, sensor housing, or the home indicator on iOS devices.
```diff
- import {SafeAreaProvider} from 'react-native-safe-area-context';
+ import {SafeAreaProvider, SafeAreaView} from 'react-native-safe-area-context';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->